test/IntegrationTests/MSTest.Acceptance.IntegrationTests/ConfigurationSettingsTests.cs

```diff
@@ -57,6 +57,41 @@ public async Task TestConfigJson_WithoutRunSettings_BuildSuccess(string tfm)
         testHostResult.AssertExitCodeIs(ExitCodes.Success);
     }
 
+    public async Task TestWithConfigFromCommandLineWithMapInconclusiveToFailedIsTrue()
+    {
+        var testHost = TestHost.LocateFrom(_testAssetFixture.ProjectPath, TestAssetFixture.ProjectName, TargetFrameworks.NetCurrent.Arguments);
+        TestHostResult testHostResult = await testHost.ExecuteAsync("--config dummyconfigfile_map.json", environmentVariables: new()
+        {
+            ["TestWithConfigFromCommandLine"] = "true",
+        });
+
+        testHostResult.AssertExitCodeIs(ExitCodes.AtLeastOneTestFailed);
+        testHostResult.AssertOutputContainsSummary(failed: 1, passed: 1, skipped: 0);
+    }
+
+    public async Task TestWithConfigFromCommandLineWithMapInconclusiveToFailedIsFalse()
+    {
+        var testHost = TestHost.LocateFrom(_testAssetFixture.ProjectPath, TestAssetFixture.ProjectName, TargetFrameworks.NetCurrent.Arguments);
+        TestHostResult testHostResult = await testHost.ExecuteAsync("--config dummyconfigfile_doNotMap.json", environmentVariables: new()
+        {
+            ["TestWithConfigFromCommandLine"] = "true",
+        });
+
+        testHostResult.AssertExitCodeIs(ExitCodes.Success);
+        testHostResult.AssertOutputContainsSummary(failed: 0, passed: 1, skipped: 1);
+    }
+
+    public async Task TestWithConfigFromCommandLineWithNonExistingFile()
+    {
+        var testHost = TestHost.LocateFrom(_testAssetFixture.ProjectPath, TestAssetFixture.ProjectName, TargetFrameworks.NetCurrent.Arguments);
+        TestHostResult testHostResult = await testHost.ExecuteAsync("--config dummyconfigfile_not_existing_file.json", environmentVariables: new()
+        {
+            ["TestWithConfigFromCommandLine"] = "true",
+        });
+
+        testHostResult.AssertStandardErrorContains("FileNotFoundException");
```

Comment:
If the name of the file is part of the output, can we capture it in the assert please.